src/containers/pages/PractitionerViews/PractitionerListView/tests/index.test.tsx

```diff
@@ -0,0 +1,190 @@
+} from '../../../../../constants';
+import store from '../../../../../store';
+import * as practitionerDucks from '../../../../../store/ducks/opensrp/practitioners';
+import * as fixtures from '../../../../../store/ducks/tests/fixtures';
+import { state } from './fixtures';
+
+// tslint:disable-next-line: no-var-requires
+const fetch = require('jest-fetch-mock');
+
+const history = createBrowserHistory();
+
+describe('src/containers/TeamAssignment/PractitionersListView/', () => {
+  beforeEach(() => {
+    store.dispatch(practitionerDucks.removePractitionersAction);
+    jest.resetAllMocks();
+  });
+
+  it('renders a dumb TeamListView correctly', () => {
+    fetch.once(JSON.stringify(fixtures.practitioners));
+    const mock: any = jest.fn();
+    const props = {
+      history,
+      location: mock,
+      match: {
+        isExact: true,
+        params: { id: '' },
+        path: PRACTITIONERS_LIST_URL,
+        url: PRACTITIONERS_LIST_URL,
+      },
+      practitioners: fixtures.practitioners,
+    };
+    const wrapper = mount(
+      <Router history={history}>
+        <PractitionersListView {...props} />
+      </Router>
+    );
+    // page title
+    const helmet = Helmet.peek();
+    expect(helmet.title).toEqual(`${PRACTITIONERS}(${fixtures.practitioners.length})`);
+
+    // should display a breadcrumb
+    expect(toJson(wrapper.find('Breadcrumb'))).toMatchSnapshot('Breadcrumb');
+
+    // should have link to add practitioner
+    expect(wrapper.find(`LinkAsButton`).length).toEqual(1);
+
+    // should have form to search practitioners
+    expect(wrapper.find('input#search').length).toEqual(1);
+
+    // should have a table
+    expect(toJson(wrapper.find('tbody tr').first())).toMatchSnapshot(
+      'First table record in listview'
+    );
+    wrapper.unmount();
+  });
+
+  it('Makes the expected calls to the opensrpService', async () => {
+    const mockList = jest.fn(async () => fixtures.practitioners);
+    const classMock = jest.fn().mockImplementation(() => {
+      return {
+        list: mockList,
+      };
+    });
+    const fetchedMock = jest.fn(arg => practitionerDucks.fetchPractitioners(arg));
+    const mock: any = jest.fn();
+    const props = {
+      fetchPractitionersCreator: fetchedMock,
+      history,
+      location: mock,
+      match: {
+        isExact: true,
+        params: { id: '' },
+        path: PRACTITIONERS_LIST_URL,
+        url: PRACTITIONERS_LIST_URL,
+      },
+      practitioners: fixtures.practitioners,
+      serviceClass: classMock,
+    };
+    mount(
+      <Router history={history}>
+        <PractitionersListView {...props} />
+      </Router>
+    );
+
+    await flushPromises();
+    expect(classMock).toBeCalledWith(OPENSRP_PRACTITIONER_ENDPOINT);
+    expect(mockList).toHaveBeenCalled();
+    expect(fetchedMock).toHaveBeenCalledWith(fixtures.practitioners);
+  });
+
+  // TODO - this is skipped, currently intentionally
+  xit('E2E flow for searching a practitioner', () => {
```

Comment:
Could we create an issue for this and remove it, please?

Reply:
yeah